### ISSUE TITLE
Feeding not working

### ISSUE THREAD

**Issue by davidcallanan** [jangxx/node-vjoy#3]
Hi, firstly thanks for creating this package.

I have completed code that a) checks if vJoy is enabled, b) creates a device, and c) updates the axes and buttons.

All the code is running without any errors.

However, only part (a) and (b) seem to be working. None of the input (part c) is appearing when I run the "Monitor vJoy" application provided by vJoy.

Any ideas why this might be the case?

If I use the "vJoy Feeder (Demo)" application provided by vJoy, the inputs are working, so it is either an issue with my code or this package.

Here is a few snippets of my code:

```
if (vJoy.isEnabled()) {
	console.log("INFO: vJoy is enabled.");
} else {
	console.log("ERROR: vJoy is either not installed or not enabled.");
	process.exit();
}
```

```
for (var i = 1; i <= 8; i++) {
	if (vJoyDevice.status(i) != "free") {
		continue;
	} else {
		break;
	}
}

player.device = vJoyDevice.create(i);

if (player.device) {
    // ...
} else {
    // ...
}
```

```
// this code does not give errors as long as I map float from range -1 to 1, to the range 1 to 32768
device.axes.X.set(Math.floor((x + 1) / 2 * 32768) + 1);
device.axes.Y.set(Math.floor((y + 1) / 2 * 32768) + 1);
```

```
device.buttons[4].set(true);
```

I am using [`v2.2.0.0` of vJoy provided by `njz3`](https://github.com/njz3/vJoy/releases/tag/v2.2.0.0).

**Comment by jangxx** [comment]
I tried to reproduce your issue and I can confirm that the feeder is not working in v2.2.0 for me as well. Unfortunately, the feeder that came with the installation doesn't work either, leaving me to the conclusion that my installation is messed up somehow. Until I get it working, I can't start working on a fix for your problem.

For now I would recommend to just install the old "official" version 2.1.9 of vJoy (or whatever the last one before 2.2.0 was), which still works fine. I will get back to you if I made some progress. Alternatively (depending on what you want to do) you could also use the ViGEmBus driver together with the vigemclient library to create virtual gamepads.

**Comment by jangxx** [comment]
Okay I got it working, and I also found out that the newer version of the SDK is apparently backwards compatible with the last official version, so simply updating the SDK to the latest version should introduce no problems. I pushed a version 1.2.0 to the repo, which you can try for yourself. If everything works as expected, I will publish the version to npm.

**Comment by davidcallanan** [comment]
😀😀😀 This is amazing! Thanks for getting back so quickly, it's working perfectly.

**Comment by jangxx** [comment]
That's great to hear! I just published version 1.2.0 to npm. Still, I really hope that you're not using this library to simulate Xinput devices/Xbox controllers, as I've seen some people do. Please use the ViGEmBus driver if you're doing Xinput stuff.
